ignition/include/gympp/gazebo/GazeboWrapper.h

```diff
@@ -22,8 +22,13 @@
 #define DEFAULT_VERBOSITY 4
 #endif
 
+#ifndef INF_RTF
+#define INF_RTF 1E9
+#endif
```

Comment:
From a fancy C++23 codebase such as this one I would have been expecting a `constexpr double` or something similar. :D

Reply:
Yeah probably it's better remove this entirely and just use `std::numeric_limits<double>::max()` as default value for the parameter.